### PR TITLE
Jetpack compose app to acceptance

### PR DESCRIPTION
**Pull request by MiKessels**
This version of the application is now fully build with Jetpack Compose (and -off course- Kotlin). 

Functional 
- Loginscreen
- Volunteerscreen
- Addrepairsreen
- Ability to add a repair, and switch between screens
- Logout and exit app

~Not functional (yet)~
- Repairmanscreens ~(in progress)~ #22

### CODE REVIEW

**Comment by MiKessels** [comment]
Core functionality is present from now on.
Bugs or future layout changes are possible. 
If possible I keep working on more features!

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ build/*
 .externalNativeBuild
```
Comment (cyrildewit) [inline .gitignore:7]: Shouldn't you exclude `.gradle` as well? These are build files. You usually don't commit these files to the repository.

File: README.md [hunk 2]
```diff
@@ -22,7 +34,8 @@ The application is built using Kotlin.
 
 
 ## Changelog
-
+_03 jan 2022_  
+Fully functional app is ready for PR to Acceptance. No logic added to app yet. So not production ready!
```
Comment (cyrildewit) [inline README.md]: If you like to maintain a changelog, you might be interested in "Keep a Changelog" (https://keepachangelog.com/en/1.0.0/)

Comment (MiKessels) [reply README.md]: Will look in to this!

File: .gitignore (hunk 1 above)
Comment (MiKessels) [reply .gitignore:7]: Thanks, didn't know this. Will resolve this in the next few days.

**Review by cyrildewit — APPROVED** [review]

**Pull request merged**